ini_multiline_value_fix_test.go

```diff
@@ -0,0 +1,60 @@
+package ini_test
+
+import (
+	"gopkg.in/ini.v1"
+	. "github.com/smartystreets/goconvey/convey"
+	"path/filepath"
+	"testing"
+)
+
+type testData struct {
+	Value1 string `ini:"value1"`
+	Value2 string `ini:"value2"`
+}
+
+func TestMultiline(t *testing.T) {
+	Value1 := "some text here\n"+
+		"some more text here\n"+
+		"\n"+
+		"there is an empty line above and below\n"
+
+	Value2 := "there is an empty line above\n"+
+		"that is not indented so it should not be part\n"+
+		"of the value"
+
+	Convey("Parse Python-style multiline values", t, func() {
+
+		// enable Python-style multiline values
+		opts := ini.LoadOptions{
+			AllowPythonMultilineValues: true,
+		}
+
+		// load test data
+		path := filepath.Join("testdata", "multiline.ini")
```

Comment:
Static test data can be defined use a `const`, unless there is a good reason that we must interact with file system.

Reply:
As a user, I find it easier to see actual examples of ini files that are parsed correctly in the `testdata` folder rather than to have to read test cases to find them. I would prefer to keep the test data in the separate ini file.

Reply:
AFA I see, the content of "multiline.ini" is too simple and straightforward, not worth keeping it in a file unless the purpose of test is to test interactions with file system (i.e. read/write a file).

Reply:
For good examples to users, documentation is the better place.

Reply:
I added another test case to cover large values. I think this should be in a separate test file.